wasm-fuzzer/wafl.sh

```diff
@@ -0,0 +1,83 @@
+currentID=$$
+
+echo "wafl ID $currentID"
+
+export DOCKER_INTERFACE_SRC=$(pwd)/wafl
+export DOCKER_AFL_INPUT=$DOCKER_INTERFACE_SRC/in-$currentID
+
+export SWAM_SOCKET_PORT=9999
+export SWAM_SOCKET_HOST=0.0.0.0
+export DOCKER_AFL_OUTPUT=out-$currentID
+
+export REQUIRED_BYTES=4
+
+
+export DOCKER_LOGS=$DOCKER_INTERFACE_SRC/out
+
+export DUMMY_TESTING_AFL=False
+
+
+export AFL="$(pwd)/aflpp/afl-fuzz"
+export LOCAL_AFL="True"
+export WASI_FILTER="True"
+export WASI=True
+export WITH_TIMEOUT="False"
+export INFER_INIT_VALUES="True"
+export INIT_VALUES_SEED="0"
+export TIMEOUT=60 # 60 seconds
+
+mkdir -p $DOCKER_AFL_INPUT
+
+## Replace env variables
+set -a
+source .env
```

Comment:
I think if you're doing this, you might as well remove half of the export statements above.